app/views/content/non-uk-teachers/tips-on-applying-for-teacher-training-as-a-non-uk-citizen.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "How to apply for teacher training in England as a non-UK citizen"
+subcategory: If you want to train to teach
+description: |-
+  Learn more about choosing a course and increasing your chances of success.
```

Comment:
I'm not sure where the description content appears? I thought it was the category content is as follows
![Screenshot 2024-06-18 at 12 09 06](https://github.com/DFE-Digital/get-into-teaching-app/assets/38726669/82fbdf70-2f06-4cab-8fc9-df5386f4dd0a)

Reply:
Ah, I thought it was the meta description - honestly not sure tho!

Reply:
`description:` is the meta description that is likely to appear in search results - so really important that it is clear that this content is for non-UK citizens

@laurahermionetennant92 It would be worth looking at the meta description for all of your pages in this PR to make sure it is descriptive and clear re non-UK (but is less than 160 characters

cc @Sarah-DfE )